auction-app-frontend/src/services/productService.js

```diff
@@ -0,0 +1,16 @@
+import { getRequest } from "src/utils/httpUtils";
+
+// fetch products paginated based on the endpoint (paginated, newArrivals, lastChance)
+export const fetchProducts = (endpoint, page, size) => {
+  return getRequest(`/products/${endpoint}?page=${page}&size=${size}`);
+};
+
+export const getProductsPaginated = (page, size) =>
```

Comment:
You can have just one export which will be object with this methods, and later you will use it like `productService.getProducts`